lib/Tracy/DIPanel/di-panel.phtml

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace MailPoet\Tracy\DIPanel;
+
+/**
+ * @var DIPanel $this
+ */
+
+?>
+<h1>Dependency Injection</h1>
+<div class="tracy-inner">
+  <h2>Free</h2>
```

Comment:
We could add a number of services into this and premium heading. `Free (55 services)`